test/e2e/lib/driver-manager.js

```diff
@@ -9,6 +9,7 @@ import config from 'config';
 import proxy from 'selenium-webdriver/proxy';
 import SauceLabs from 'saucelabs';
 import { times } from 'lodash';
+import fs from 'fs';
```

Comment:
Could we import only `readFileSync` as it's the only function that we use from the `fs` module?

Reply:
Yep, that works. thanks for the suggestion!